src/content/topics/integrations/sleuth.mdx

```diff
@@ -0,0 +1,94 @@
+     </CalloutDescription>
+   </Callout>
+
+![LaunchDarkly authorization dialog](../images/sleuth-ld-integration-auth-dialog.png)
+
+5. The integration is successful when the green **LaunchDarkly feature flags enabled** button is displayed in the Sleuth integration card for Feature Flags. 
+
+![A successful integration!](../images/sleuth-ld-integration-success-dialog_3.png)
+
+## Configuring the Sleuth integration
+
+Now that the Sleuth LaunchDarkly integration has been set up, you must now configure Sleuth to know which feature flags it should track. 
+
+To select a LaunchDarkly project and environment to track:  
+
+1. Click the **LaunchDarkly feature flags enabled** button (shown above). 
+
+2. Select the **Feature Flag Project** and **Environment** you would like to track. The projects and environments shown in the dropdowns are taken from your LaunchDarkly configuration.  
+
+![Feature flag change source setup in Sleuth](../images/sleuth-ld-integration-config-dialog.png)  
+
+![Projects and environments in LaunchDarkly](../images/sleuth-ld-integration-projects-dialog.png)
+
+3. Give the integration a **Name**, then press **Create**. 
+
+4. Once the connection is established, you will be taken to the Sleuth trend graph, where you will be able to monitor how your feature flags impact your code over time. You're set&mdash;now sit back and let Sleuth track your code over time. 
+
+![Sleuth trend graph](../images/sleuth-ld-integration-trendgraph.png)
+
+<Callout intent="info">
+  <CalloutTitle>Tracking multiple projects/environments</CalloutTitle>
+    <CalloutDescription>
+       To track multiple LaunchDarkly projects/environments in Sleuth, create a new Sleuth project. You can create as many Sleuth projects as needed to track any number of feature flag project and environment combinations. 
+       
+       For more information, read <a href="https://help.sleuth.io/projects#creating-a-project">Creating a project</a> in Sleuth Documentation.  
+  </CalloutDescription>
+</Callout>
+
+## Disabling the Sleuth integration
+
+To disable the Sleuth integration:
+
+1. Click the gear icon in the upper-right corner of the dashboard. 
+2. Click **Delete**. Click **Delete** again if you're sure. 
+
+![Deleting the LaunchDarkly integration in Sleuth](../images/sleuth-ld-integration-delete_2.png)
+
+3. The integration is removed.
+
+## LaunchDarkly Slack notifications from Sleuth
+
+With Slack integration in Sleuth, notify your entire team or just the commit author/PR initiator of a code change, including feature flag changes. 
+
+For more information, read about the <a href="https://help.sleuth.io/integrations/slack">Slack integration</a> in Sleuth Documentation. The Sleuth Slack integration works independently&mdash;yet alongside&mdash;the [LaunchDarkly Slack integration](/integrations/slack/). 
```

Comment:
@mtbnut Out of curiosity, have you tested your integration in a Slack team that is also using the LaunchDarkly Slack app? Do you receive duplicate notifications from both when flags that are associated with the Sleuth integration change?

Reply:
@scribblingfox The same event could trigger multiple messages through each integration (i.e., Sleuth and LD). However, they would be easily discernible, as they'd be styled differently and more than likely end up in separate Slack channels.